### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by leefsmp** [netzwerg/react-svg-timeline#21]
I know I can just dig the example code and figure it out by myself ... but a little explicit doc about the supported props and format would not hurt. Thanks.

**Comment by kashifshamaz21** [comment]
@netzwerg I really like this library for rendering event timelines & would love to integrate it in my web-app. But to get started, there doesn't seem to be any guiding docs or starting point other than the example source code. I looked at the example, and felt like the starting curve is quite a bit, since the internals of the library need to also be understood to work our way through the example and use it in our custom requirements.
Any help in the documentation or usage of the component would be very useful and much appreciated. :) Thanks in advance.

**Comment by netzwerg** [comment]
I agree @kashifshamaz21, docs are still missing, and the example is actually an illustration of all the possibilities, and not an easy way to get started!

This is how you could use the component in the simplest possible way:

```jsx
export const App = () => {
  const laneId = 'demo-lane'
  const lanes = [
    {
      laneId,
      label: 'Demo Lane',
    },
  ]
  const events = [
    {
      eventId: 'event-1',
      laneId: laneId,
      startTimeMillis: 1399845600000,
    },
    {
      eventId: 'event-2',
      laneId,
      startTimeMillis: 1167606000000,
      endTimeMillis: 1230698892000,
    },
  ]
  return (
    <Timeline width={600} height={300} events={events} lanes={lanes} dateFormat={(ms) => new Date(ms).toString()} />
  )
}
```

Hope this helps!

**Comment by netzwerg** [comment]
@TD-DO I agree 😬 Maybe the [type definition](https://github.com/netzwerg/react-svg-timeline/blob/bc317f9efbaac5715037f193a93471e1cefee469/src/Timeline.tsx#L24) can point in the right direction.
